pxt.json

```diff
@@ -35,7 +37,8 @@
     "yotta": {
         "config": {
             "codal": {
-                "MLRUNNER_USE_EXAMPLE_MODEL": 0
+                "MLRUNNER_USE_EXAMPLE_MODEL": 1,
+                "ML_DEBUG_PRINT": 1
```

Comment:
Ultimately we need to remove these so that the user can specify them (if project tries to overwrite an extension configuration it throws an error), but for now these can be used for testing.